config/optional/views.view.civiremote_funding_application_list.yml

```diff
@@ -1129,15 +1090,15 @@ display:
               empty_column: false
               responsive: ''
             application_process_id:
-              sortable: false
+              sortable: true
```

Comment:
`sortable: true` may has been introduced by another view feature as well. shall it be reverted to `false`?